Description: currently we don't use the outloud server in the Debian package,
 but it requires g++_multilib to build, which would be weird to do in the
 arch-indep target. So don't install it and there is no issue.
Author: Paul Gevers <elbrus@debian.org>
Forwarded: no

Index: emacspeak/Makefile
===================================================================
--- emacspeak.orig/Makefile
+++ emacspeak/Makefile
@@ -254,7 +254,6 @@ install:
 	$(CP) -r $(PHANTOM) $(libdir)/phantom
 	$(INSTALL) -d $(libdir)/servers
 	$(INSTALL) -d $(libdir)/servers/linux-outloud
-	$(INSTALL)  -m 755 ${OUTLOUD}  $(libdir)/servers/linux-outloud
 	$(INSTALL) -d $(libdir)/servers/linux-espeak
 	$(INSTALL)  -m 755 ${ESPEAK}  $(libdir)/servers/linux-espeak
 	$(INSTALL) -d $(libdir)/servers/software-dtk
